lib/sms.js

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+var Nexmo = require('nexmo')
+var P = require('bluebird')
+
+module.exports = function (log, translator, templates, config) {
+  var nexmo = new Nexmo({
+    apiKey: config.apiKey,
+    apiSecret: config.apiSecret
+  })
+  var sendSms = P.promisify(nexmo.message.sendSms, { context: nexmo.message })
+
+  var LINK = 'http://mzl.la/1HOd4ec'
+
+  return {
+    send: function (phoneNumber, senderId, messageId, acceptLanguage) {
+      log.info({
+        op: 'sms.send',
+        phoneNumber: phoneNumber,
+        senderId: senderId,
+        messageId: messageId,
+        acceptLanguage: acceptLanguage
+      })
+
+      return P.resolve()
+        .then(function () {
+          var message = getMessage(messageId, acceptLanguage)
+
+          return sendSms(senderId, phoneNumber, message.trim())
+        })
+        .then(function (result) {
+          var resultCount = result.messages && result.messages.length
+          if (resultCount !== 1) {
+            log.error({ op: 'sms.send', err: new Error('Unexpected result count'), result: result })
```

Comment:
we might benefit from a "success!" log here with either `.info` (or trace). I think we already use `.info` for successful Push notifications